File: CSHARP/Cap03/c03ex05.cs
// c03ex05.cs

using System;

public class Program
{
    public static void Main(string[] args)
    {
        string nome;

        Console.Write("Entre seu nome: ");
        nome = Console.ReadLine();

        Console.WriteLine($"Ola, {nome}! Bem-vindo ao estudo de C#.");

        Console.Write("\nAperte <Enter> para encerrar... ");
        Console.ReadLine();
    }
}

File: KOTLIN/Cap03/c03ex05.kt
// c03ex05.kt

fun main() {
    var name: String

    print("Entre seu nome: ")
    name = readLine() ?: ""

    println("Ola, $name! Bem-vindo ao estudo de Kotlin.")

    print("\nAperte <Enter> para encerrar... ")
    readLine()
}
